integration-tests/pict-models/default.pict

```diff
@@ -0,0 +1,16 @@
+# This is file used for integration tests (PR checks, post-merge checks)
+#
+
+# OCP: 4.17, 4.16
+# ACS: valid values:  new, hosted
+# Registry: valid values:  quay, quay.io, artifactory
+# TPA: valid values:  new, hosted
+# SCM: valid values:  github, gitlab, bitbucket
+# Pipeline: valid values:  github, gitlab, bitbucket
+
+OCP: 4.17
```

Comment:
https://github.com/redhat-appstudio/rhtap-cli/pull/366/files Has also `Auth`. Is it not required or is it missing? 
(It's also missing in the commented section documenting possible values (in rhtap-cli as well)

Reply:
It's missing, `Auth` is added.